site/docs/guides/operations/kubernetes/healthchecks.md

```diff
@@ -0,0 +1,257 @@
+HTTP/1.1 200
+Connection: keep-alive
+Content-Length: 42
+Content-Type: text/plain;charset=UTF-8
+Date: Thu, 18 May 2023 17:10:27 GMT
+Keep-Alive: timeout=60
+
+Egeria OMAG Server Platform (version 4.0)
+
+
+
+### Server
+This API call was introduced in [release 3.4](/release-notes/3-4.md)
+
+#### Server is not known
+```
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/servers/cocoMDS99/instance/status"
+HTTP/1.1 200
+Connection: keep-alive
+Content-Type: application/json
+Date: Thu, 18 May 2023 17:08:15 GMT
+Keep-Alive: timeout=60
+Transfer-Encoding: chunked
+
+{
+    "actionDescription": "getActiveServerStatus",
+    "class": "OMAGServerStatusResponse",
+    "exceptionClassName": "org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException",
+    "exceptionErrorMessage": "OMAG-MULTI-TENANT-404-001 The OMAG Server cocoMDS99 is not available to service a request from user admin",
+    "exceptionErrorMessageId": "OMAG-MULTI-TENANT-404-001",
+    "exceptionErrorMessageParameters": [
+        "cocoMDS99",
+        "admin"
+    ],
+    "exceptionProperties": {
+        "parameterName": "serverName",
+        "serverName": "cocoMDS99"
+    },
+    "exceptionSystemAction": "The system is unable to process the request because the server is not running on the called platform.",
+    "exceptionUserAction": "Verify that the correct server is being called on the correct platform and that this server is running. Retry the request when the server is available.",
+    "relatedHTTPCode": 404
+}
+
+```
+#### No permission for api call
+```
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/servers/cocoMDS5/instance/status"
+HTTP/1.1 200
+Connection: keep-alive
+Content-Type: application/json
+Date: Thu, 18 May 2023 17:07:28 GMT
+Keep-Alive: timeout=60
+Transfer-Encoding: chunked
+
+{
+"actionDescription": "validateUserForServer",
+"class": "OMAGServerStatusResponse",
+"exceptionClassName": "org.odpi.openmetadata.commonservices.ffdc.exceptions.UserNotAuthorizedException",
+"exceptionErrorMessage": "OMAG-PLATFORM-SECURITY-403-002 User admin is not authorized to issue a request to server cocoMDS5",
+"exceptionSystemAction": "The system is unable to process a request from the user because they do not have access to the requested OMAG server.  The request fails with a UserNotAuthorizedException exception.",
+"exceptionUserAction": "Determine whether the user should have access to the server.  If they should have, take steps to add them to the authorized list of users.  If this user should not have access, investigate where the request came from to determine if the system is under attack, or it was a mistake, or the user's tool is not configured to connect to the correct server.",
+"relatedHTTPCode": 403
+}
+```
+### Server is available
+```
```

Comment:
There is also a simpler call to test if the server is active ... `{baseURL}/open-metadata/platform-services/users/{user}/server-platform/servers/{serverName}/is-active`